__tests__/pages/webhooks/checkout-calculate-taxes.test.ts

```diff
@@ -48,14 +75,22 @@ describe("api/webhooks/checkout-calculate-taxes", () => {
     const checkoutPayload = dummyFetchTaxesPayload;
     req.body = [checkoutPayload];
 
-    // @ts-ignore
-    await handler(req, res);
+    await calculateTaxes.default(req, res);
 
     expect(res.statusCode).toBe(400);
-    expect(mockedFetchTaxesForCheckout).not.toHaveBeenCalled();
+    expect(post).not.toHaveBeenCalled();
   });
 
-  it("rejects when saleor signature is empty", async () => {
+  it.skip("rejects when saleor signature is empty", async () => {
```

Comment:
Test skipped

```suggestion
  it("rejects when saleor signature is empty", async () => {
```

Reply:
As above. It is intended. I don't check signature yet